docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  postgres:
+    image: isac:1.0
+    build:
+      context: ./
+      dockerfile: Dockerfile_database
+    container_name: 'isac'
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=ISAC_ADMIN
+      - POSTGRES_PASSWORD=pass
+      - POSTGRES_DB=ISAC
```

Comment:
please add new lines in your files
-- we need a pre-commit check for this

Reply:
new lines in which file?

Reply:
in this file, see the error
![image](https://github.com/isac-nitkkr/node-modules/assets/49202620/88a27b69-920b-4af0-a985-a5ce2e5e4594)

Reply:
nvm got it

Reply:
is this error? Or a convention?

Reply:
yes it is convention for text based files, file should end with a new line